.github/workflows/release_publish.yaml

```diff
@@ -0,0 +1,189 @@
+        echo '<settings>
+            <servers>
+              <server>
+                <id>maven-internal</id>
+                <username>${{ secrets.packages-auth-user }}</username>
+                <password>${{ secrets.packages-auth-token }}</password>
+              </server>
+              <server>
+                <id>maven-private</id>
+                <username>${{ secrets.packages-auth-user }}</username>
+                <password>${{ secrets.packages-auth-token }}</password>
+              </server>
+              <server>
+                <id>nuxeo-studio</id>
+                <username>${{ secrets.connect-auth-user }}</username>
+                <password>${{ secrets.connect-auth-token }}</password>
+              </server>
+            </servers>
+            </settings>' > ~/.m2/settings.xml
+
+    - name: Tag and check if compiling
+      run: |
+        find ./ -type f -exec sed -i '' -e "s/${{ github.event.inputs.version }}-SNAPSHOT/${{ github.event.inputs.version }}/" {} \;
+        git config user.name github-actions
+        git config user.email github-actions@github.com
+        git checkout -b release-v${{ github.event.inputs.version }}
+        git commit -a -m "Release ${{ github.event.inputs.version }}"
+        git tag -a v${{ github.event.inputs.version }} -m "Release ${{ github.event.inputs.version }}"
+        mvn -ntp install -DskipInstall
+        git push origin v${{ github.event.inputs.version }}
+        git push origin release-v${{ github.event.inputs.version }}
+
+    - name: Archive packages
+      uses: actions/upload-artifact@v2
+      with:
+        name: packages
+        path: |
+          nuxeo-ai-core-package/target/nuxeo-ai-core-${{ github.event.inputs.version }}.zip
+
+    - name: Publish Nuxeo packages
+      env:
+        CONNECT_PREPROD_URL: https://nos-preprod-connect.nuxeocloud.com/nuxeo
+      run: |
+        PACKAGE="nuxeo-ai-core-package/target/nuxeo-ai-core-${{ github.event.inputs.version }}.zip"
+        STATUS_CODE=`curl -i --silent --output publish-req.output -w "%{http_code}" -u "${{ secrets.CONNECT_PREPROD_AUTH }}" -F package=@$PACKAGE "$CONNECT_PREPROD_URL/site/marketplace/upload?batch=true"`
+        cat publish-req.output
+        if [[ "$STATUS_CODE" != "200" ]]
+        then
+          exit 1
+        else
+          exit 0
+        fi
+
+    - name: Bump release version
+      id: bump_version
+      uses: christian-draeger/increment-semantic-version@1.0.2
+      with:
+        current-version: ${{ github.event.inputs.version }}
+        version-fragment: ${{ github.event.inputs.increment }}
+
+    - name: Post Release
+      run: |
+        git checkout ${{ github.ref }}
+        find ./ -type f -exec sed -i '' -e "s/${{ github.event.inputs.version }}-SNAPSHOT/${{ steps.bump_version.outputs.next-version }}-SNAPSHOT/" {} \;
+        git add .
+        git commit -m "Post Release ${{ github.event.inputs.version }}"
+        git push ${{ github.ref }}
+
+  release-publish-lts:
```

Comment:
Seems that it is identical to 10.10 the only difference is java version, right? It might be one of the default parameters . It will also let us release from a branch